content/intro-to-storybook/ember/en/data.md

```diff
@@ -71,189 +82,87 @@ export const reducer = (state, action) => {
   }
 };
 
-export default combineReducers({
-  reducer,
-});
+export default reducers;
 ```
 
-Then we'll update our `TaskList` to read data out of the store. First let's move our existing presentational version to `app/components/pure-task-list.hbs` and `app/components/pure-task-list.js` and wrap it with a container.
-
-In `app/components/PureTaskList.hbs`:
-
-```handlebars
-{{!--app/components/pure-task-list.hbs --}}
-{{#if @loading}}
- <LoadingRow />
- <LoadingRow />
- <LoadingRow/>
- <LoadingRow />
- <LoadingRow />
-{{else if this.tasksInOrder}}
-  {{#each this.tasksInOrder as |task|}}
-    <Task
-      @task={{task}}
-      @pin={{fn @pinTask task.id}}
-      @archive={{fn @archiveTask task.id}}
-    />
-  {{/each}}
-{{else}}
-  <div class="list-items">
-    <div class="wrapper-message">
-      <span class="icon-check" />
-      <div class="title-message">You have no tasks</div>
-      <div class="subtitle-message">Sit back and relax</div>
-    </div>
-  </div>
-{{/if}}
+And finally our store (watch the import):
+
+```js
+// app/store.js
+import { createStore } from 'tracked-redux';
+import reducers from './reducers';
+
+export const store = createStore(reducers);
 ```
 
-In `app/components/task-list.hbs`:
+First we’ll construct a simple Redux store that responds to actions that change
+the state of tasks, in a file called `reducers/index.js` (intentionally kept
+simple):
 
-```handlebars
+## Using a Route
 
-{{!!-- app/components/task-list.hbs --}}
-<div>
-  <PureTaskList
-    @tasks={{this.tasks}}
-    @pinTask={{this.pinTask}}
-    @archiveTask={{this.archiveTask}}
-  />
-</div>
+With that we are able to connect our route with the redux store. Thanks to
+tracked-redux we have no artificial properties put on the prototype of our
+objects, instead we can explicitely declare each field we want. Here we are
+going to use the route to load the data and a controller with actions to modify
+it:
```

Comment:
@gossi we could phrase it a bit different if you don't mind. Probably something like:

"We have our redux store setup. Thanks to the package we can expand it as we see fit. Now we're going to use both a [route](https://guides.emberjs.com/release/routing/defining-your-routes/) and a [controller](https://guides.emberjs.com/release/routing/controllers/), the latter will contain the actions we've created earlier so that we can modify our store with relative ease.

Inside your app folder, create a new one called `tasks` and inside add a new file called `route.js` with the following:". What you think?